### PR TITLE
License, README, favicon, resources!

### PR DESCRIPTION
**Pull request by crnicholson**
cc @jaspermayone 

(Prettier made it quirky, btw)

### CODE REVIEW

**Comment by crnicholson** [comment]
I also just added a Hack Club link by clicking the flag

File: README.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-- `react-wrap-balancer` - for balancing text widths for titles and paragraphs
-- `tailwind-merge` - for merging Tailwind class names
-
-&copy; 2021 Sarthak Mohanty. Licensed under the [Unlicense](LICENSE) from Mozilla.
+# Ham Club - A New YSWS from Hack Club
+Ham Club is a program for teens in the US running from April 29th to May 31st, allowing youth to get their ham radio license and then receive a free ham radio!
+
+# You Ship, We Ship
+
+At Hack Club we're re-imagining a new type of public education for technology. We believe people learn best by creating things they care about and sharing them with others. We build open source creative tools with young people all around the world. These tools are gateways to new subjects in technology like [embedded systems](https://sprig.hackclub.com), [circuits](https://onboard.hackclub.com) or digital fabrication robotics. When teenagers build things with our tools we send them more creative materials for free.
```
Comment (jaspermayone) [inline README.md:6]: @crnicholson I commented on this before... is this the official YSWS wording for marketing? It feels very AI written right now.

File: README.md [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-
-&copy; 2021 Sarthak Mohanty. Licensed under the [Unlicense](LICENSE) from Mozilla.
+# Ham Club - A New YSWS from Hack Club
+Ham Club is a program for teens in the US running from April 29th to May 31st, allowing youth to get their ham radio license and then receive a free ham radio!
+
+# You Ship, We Ship
+
+At Hack Club we're re-imagining a new type of public education for technology. We believe people learn best by creating things they care about and sharing them with others. We build open source creative tools with young people all around the world. These tools are gateways to new subjects in technology like [embedded systems](https://sprig.hackclub.com), [circuits](https://onboard.hackclub.com) or digital fabrication robotics. When teenagers build things with our tools we send them more creative materials for free. 
+
+Our goal is to build a new type of library online that offers learners not just media to consume but tools to create. And it is free and open source to all. We're just getting started...
```
Comment (jaspermayone) [inline README.md:8]: See above comment

File: components/head.js [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+      <title>{title}</title>
+      <meta name="description" content={description} />
+      <link rel="icon" href={iconUrl} />
+      {url ? <meta property="og:url" content={url} /> : ""}
+      <meta property="og:type" content="website" />
+      <meta property="og:title" content={title} />
+      <meta property="og:description" content={description} />
+      {children}
+    </Head>
+  );
```
Comment (jaspermayone) [inline components/head.js:23]: plz fix weird formatting!

File: pages/help.js [hunk 4]
```diff
@@ -55,7 +56,7 @@ export default function Help() {
           </li>
         </div>
         <h1 className="text-2xl text-white">
-          <u>Misc:</u>
+          <u>Miscellaneous:</u>
```
Comment (jaspermayone) [inline pages/help.js]: This can still be abbreviated as `misc` and should be plz

File: pages/help.js [hunk 5]
```diff
@@ -68,7 +69,7 @@ export default function Help() {
           </li>
         </div>
         <h1 className="text-2xl text-white">
-          <u>After you study:</u>
+          <u>After You Study:</u>
```
Comment (jaspermayone) [inline pages/help.js]: Doesn't need to be capitalized.

File: pages/help.js [hunk 6]
```diff
@@ -79,11 +80,26 @@ export default function Help() {
...
-                FCC FRN Registration
-              </a>{" "}
-              - Once you finish studying, you will need to get an FRN from the FCC in order to start the test in the first place.
-              You will need to register with the FCC in order to do this!
+            <a
+              href="https://apps.fcc.gov/cores/userLogin.do?csfrToken="
+              className="underline"
+            >
+              FCC FRN Registration
+            </a>{" "}
```
Comment (jaspermayone) [inline pages/help.js:89]: weird formatting here, pls fix! I would suggest running prettier on your code before PRing.

File: pages/index.js [hunk 7]
```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
...
+
+      <div>
+        <a class="banner mx-auto ml-2 z-50" href="https://hackclub.com/">
+          <img
+            src="https://contribute.hackclub.com/images/flag-orpheus-top.png"
+            alt="Hack Club Logo"
+          ></img>
+        </a>
+      </div>
+
```
Comment (jaspermayone) [inline pages/index.js:26]: confused on why the switch was made here, the old code works fine!

File: pages/index.js [hunk 8]
```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
...
+        <section id="hero" className="flex flex-col items-center relative">
           <div className="my-24 flex flex-col gap-4 items-center text-center z-10">
-            <h1 className="text-9xl">
+            <h1 className="text-6xl sm:text-9xl">
               Ham
               <br />
               Club
             </h1>
-            <p className="text-4xl">
+            <p className="text-2xl sm:text-4xl">
```
Comment (jaspermayone) [inline pages/index.js:35]: Can you explain why this change is needed?

File: pages/index.js [hunk 9]
```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
...
       <main>
-        <section
-          id="hero"
-          className="flex flex-col items-center relative"
-        >
-          {/* <Nav /> */}
+        <section id="hero" className="flex flex-col items-center relative">
           <div className="my-24 flex flex-col gap-4 items-center text-center z-10">
-            <h1 className="text-9xl">
+            <h1 className="text-6xl sm:text-9xl">
```
Comment (jaspermayone) [inline pages/index.js:30]: Can you explain why this change is needed?

File: pages/index.js [hunk 10]
```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
...
+              Get your radio! 📻
             </a>
           </div>
         </section>
         <section className="h-fit py-16 w-screen">
-          <div id="grid" className="grid grid-cols-1 lg:grid-cols-2 lg:w-2/3 mx-auto gap-5 items-stretch justify-center *:text-black *:p-6 *:grow p-5 *:rounded-lg *:bg-white *:flex *:flex-col *:text-center *:items-center *:justify-center">
+          <div
+            id="grid"
+            className="grid grid-cols-1 lg:grid-cols-2 lg:w-2/3 mx-auto gap-5 items-stretch justify-center *:text-black *:p-6 *:grow p-5 *:rounded-lg *:bg-white *:flex *:flex-col *:text-center *:items-center *:justify-center"
+          >
```
Comment (jaspermayone) [inline pages/index.js:53]: Formatting.

File: pages/index.js [hunk 11]
```diff
@@ -2,36 +2,55 @@ import HeadObject from "../components/head";
...
               <br />
               get a free radio
             </p>
-            <a href="https://forms.hackclub.com/t/4xrbpF32B8us" target="_blank" className="uppercase px-4 py-2 rounded-full border-white border-2 text-center hover:bg-white hover:text-black">
-             Get your radio! 📻
+            <a
+              href="https://forms.hackclub.com/t/4xrbpF32B8us"
+              target="_blank"
+              className="uppercase px-4 py-2 rounded-full border-white border-2 text-center hover:bg-white hover:text-black"
+            >
```
Comment (jaspermayone) [inline pages/index.js:44]: Formatting.

File: pages/index.js [hunk 12]
```diff
@@ -47,16 +66,17 @@ export default function Home() {
               </p>
             </div>
             <div>
-              <h2>License Grant</h2>
+              <h2>Radio Grant</h2>
               <p>
-                If you pass the test by May 31st, we will cover your exam and
-                license fees!
+                If you pass the test by May 31st, we will cover your first
+                radio! Note that Hack Club will not cover the $5 exam fee.
```
Comment (jaspermayone) [inline pages/index.js]: Copytext fix:

`If you pass your test by May 31st, we'll send you a ham radio! (Note: this is for people under Middle/High Schoolers only!)`


_We don't need the part about the exam fee._

File: pages/index.js [hunk 13]
```diff
@@ -47,16 +66,17 @@ export default function Home() {
...
               </p>
             </div>
             <div>
-              <h2>UV-K5 Radio</h2>
+              <h2>UV-K6 Radio</h2>
+              {/* This is a bit repetitive! Should we combine the one before this with this one? */}
```
Comment (jaspermayone) [inline pages/index.js:77]: I think it's fine

File: pages/index.js [hunk 14]
```diff
@@ -73,11 +93,30 @@ export default function Home() {
                 Want to jump right in? Check out our{" "}
                 <a href="/help" className="underline">
                   list of ham radio resources and FAQs!
+                </a>{" "}
+                Want to learn about the application process? Check out our{" "}
+                <a href="/process" className="underline">
+                  guide
```
Comment (jaspermayone) [inline pages/index.js]: Please remove this, it isn't needed.

File: pages/process.js [hunk 15]
```diff
@@ -0,0 +1,159 @@
+import Link from "next/link";
+
```
Comment (jaspermayone) [inline pages/process.js]: Please combine the info on this page into the FAQ.

Comment (jaspermayone) [reply pages/process.js]: I don't see the need for this to be a whole separate file/page, it just adds complexity.

File: public/ham1.jpg
Comment (jaspermayone) [inline public/ham1.jpg:1]: confused on why this was added. if it was needed, please let me know why.

File: public/ham2.jpg
Comment (jaspermayone) [inline public/ham2.jpg:1]: confused on why this was added. if it was needed, please let me know why.

File: styles/globals.css [hunk 16]
```diff
@@ -44,4 +44,36 @@ div#grid > div {
...
+
+@keyframes wave-flag {
+  0% {
+    transform: rotate(0deg);
+  }
+
+  100% {
+    transform: rotate(-5deg);
+  }
+}
```
Comment (jaspermayone) [inline styles/globals.css:79]: let's use tailwind whenever possible plz

**Review by jaspermayone — CHANGES_REQUESTED** [review]
Left some comments. In general, I would suggest running prettier on your code before PR'ing. Please respond / mark as resolved as you make the changes so I am not doing a full review every time, as this was part of what blocked me on making my review on this for so long.

File: README.md (hunk 1 above)
Comment (crnicholson) [reply README.md:6]: Yep this is from Blot.

File: components/head.js (hunk 3 above)
Comment (crnicholson) [reply components/head.js:23]: This is what Prettier did!

File: pages/index.js (hunk 7 above)
Comment (crnicholson) [reply pages/index.js:26]: To make it actually work, because it didn't work before.

File: pages/index.js (hunk 8 above)
Comment (crnicholson) [reply pages/index.js:35]: For mobile

File: pages/help.js (hunk 6 above)
Comment (crnicholson) [reply pages/help.js:89]: this is what Prettier did

File: pages/index.js (hunk 9 above)
Comment (crnicholson) [reply pages/index.js:30]: to make it work for mobile

File: README.md (hunk 1 above)
Comment (jaspermayone) [reply README.md:6]: Where? I don't see it?

File: pages/index.js (hunk 10 above)
Comment (crnicholson) [reply pages/index.js:53]: this is what prettier did 🤷

File: pages/index.js (hunk 11 above)
Comment (crnicholson) [reply pages/index.js:44]: this is what prettier did 🤷

File: public/ham2.jpg
Comment (crnicholson) [reply public/ham2.jpg:1]: for the favicon

File: public/ham1.jpg
Comment (crnicholson) [reply public/ham1.jpg:1]: for the favicon

File: pages/help.js (hunk 5 above)
Comment (crnicholson) [reply pages/help.js]: To look good with the rest of the page, it should be.

File: public/ham2.jpg
Comment (jaspermayone) [reply public/ham2.jpg:1]: It doesn't look super professional to use a ham (the meat) for the favicon, and we also don't need 2 files. Please PR a ham radio or some more appropriate favicon!

File: public/ham1.jpg
Comment (jaspermayone) [reply public/ham1.jpg:1]: It doesn't look super professional to use a ham (the meat) for the favicon, and we also don't need 2 files. Please PR a ham radio or some more appropriate favicon!

File: pages/index.js (hunk 9 above)
Comment (jaspermayone) [reply pages/index.js:30]: It seemed to work on mobile before, do you have screenshots of it not working?

File: pages/index.js (hunk 8 above)
Comment (jaspermayone) [reply pages/index.js:35]: It seemed to work on mobile before, do you have screenshots of it not working?

File: pages/help.js (hunk 5 above)
Comment (jaspermayone) [reply pages/help.js]: I don't actually think this is the case... we have other non caps text.

File: pages/index.js (hunk 12 above)
Comment (crnicholson) [reply pages/index.js]: Then I think we're being a bit misleading.

File: pages/process.js (hunk 15 above)
Comment (crnicholson) [reply pages/process.js]: but the FAQ page shouldn't have it because it is an FAQ page.

Comment (crnicholson) [reply pages/process.js]: It's too much information for one page

File: pages/index.js (hunk 12 above)
Comment (jaspermayone) [reply pages/index.js]: Nowhere else do we say we cover the exam fee.

File: styles/globals.css (hunk 16 above)
Comment (crnicholson) [reply styles/globals.css:79]: Tailwind is powerful but it can't do animations and stuff like that

Comment (crnicholson) [reply styles/globals.css:79]: At least that's what I think

File: README.md (hunk 1 above)
Comment (crnicholson) [reply README.md:6]: image

https://github.com/hackclub/blot

File: pages/help.js (hunk 5 above)
Comment (crnicholson) [reply pages/help.js]: Not if you scroll up a bit

File: pages/index.js (hunk 12 above)
Comment (crnicholson) [reply pages/index.js]: We should let people know that this isn't free

File: public/ham1.jpg
Comment (crnicholson) [reply public/ham1.jpg:1]: I have no idea what I'm looking for. Could you give a more specific example?

File: pages/index.js (hunk 8 above)
Comment (crnicholson) [reply pages/index.js:35]: image
this isn't ideal

File: pages/process.js (hunk 15 above)
Comment (crnicholson) [reply pages/process.js]: if you have a better way of going about this let me know

File: pages/help.js (hunk 5 above)
Comment (crnicholson) [reply pages/help.js]: It just looks weird on the rest of the page with all the other capilizations

File: pages/index.js (hunk 8 above)
Comment (jaspermayone) [reply pages/index.js:35]: what does it look like once you added that? and does it affect the larger version of the site?

File: pages/index.js (hunk 12 above)
Comment (jaspermayone) [reply pages/index.js]: I still firmly think this isn’t needed.

File: pages/process.js (hunk 15 above)
Comment (jaspermayone) [reply pages/process.js]: I don't think we need another page, please combine / reduce the info so it can be added to FAQ.

File: public/ham1.jpg
Comment (jaspermayone) [reply public/ham1.jpg:1]: We want a favicon that isn't a photo of a ham as that isn't really on brand

Comment (jaspermayone) [reply public/ham1.jpg:1]: perhaps something like a ham radio or some other radio related thing would be a more appropriate favicon

File: pages/process.js (hunk 15 above)
Comment (crnicholson) [reply pages/process.js]: Done

File: public/ham1.jpg
Comment (crnicholson) [reply public/ham1.jpg:1]: Done

File: public/ham2.jpg
Comment (crnicholson) [reply public/ham2.jpg:1]: Done

File: pages/help.js (hunk 4 above)
Comment (crnicholson) [reply pages/help.js]: Done

File: pages/index.js (hunk 8 above)
Comment (crnicholson) [reply pages/index.js:35]: image
It does not affect the size on the larger version of the site.

File: pages/index.js (hunk 9 above)
Comment (crnicholson) [reply pages/index.js:30]: See above.

File: pages/index.js (hunk 12 above)
Comment (crnicholson) [reply pages/index.js]: Fixed

File: pages/index.js (hunk 14 above)
Comment (crnicholson) [reply pages/index.js]: done

**Review by jaspermayone — COMMENTED** [review]
LGTM, just testing deploy

**Comment by jaspermayone** [comment]
argh deploys are protected

**Comment by jaspermayone** [comment]
just gonna merge it

**Review by jaspermayone — APPROVED** [review]
